packages/@sanity/components/src/edititem/PopOver.js

```diff
@@ -21,248 +25,269 @@ export default class EditItemPopOver extends React.Component {
       handleClick: PropTypes.func
     })),
     fullWidth: PropTypes.bool,
-    onNeedScroll: PropTypes.func,
+    isOpen: PropTypes.bool,
     scrollContainer: PropTypes.node,
-    scrollContainerId: PropTypes.string,
-    isOpen: PropTypes.bool
-  }
+    scrollContainerId: PropTypes.string
+  };
 
   static defaultProps = {
     onClose() {}, // eslint-disable-line
-    scrollContainerId: 'Sanity_Default_DeskTool_Editor_ScrollContainer',
     actions: [],
-    isOpen: true
+    isOpen: true,
```

Comment:
Maybe not related to these changes, but can we do without this flag? Usually its better do not render the component at all if it shouldn't be open.